src/anm/graphics/brush.js

```diff
@@ -406,9 +406,18 @@ Brush.clearShadow = function(ctx) {
  * @param {anm.Brush} return.return a brush value as a result of interpolation
  */
 Brush.interpolateBrushes = function(from, to) {
+    var equal = is.equal(from, to);
     from = (from instanceof Brush) ? from : Brush.value(from);
-    to   = (to   instanceof Brush) ? to   : Brush.value(to);
     if (!from._converted) { from.convertColorsToRgba(); }
+    if (equal) {
+        //if the values are the same, we can just skip the interpolating
+        //and return the first value
+        return function() {
```

Comment:
Also I think there's too much of comments here and there, please do not comment obvious things, there should be a balance :)